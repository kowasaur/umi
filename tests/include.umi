include "./tests/include/other.umi"
include "./tests/include/cs.cs"

il class CS "CS" {
    static ilf Void func() "Func"
}

Void main() {
    otherFileFunction()
    CS.func()
}
