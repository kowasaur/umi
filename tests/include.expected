this was defined in another file
this was written in C#
